src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -71,5 +71,11 @@ def load_arguments(self, _):
         c.argument('container_name', help='The container name to tail the logs. If omitted, the first container in the container group will be chosen')
         c.argument('follow', help='Indicate to stream the tailing logs', action='store_true')
 
+    with self.argument_context('container exec') as c:
+        c.argument('container_name', help='The container name to exec the command. If omitted, the first container in the container group will be chosen')
```

Comment:
Regarding the default, is this an arbitrary choice?

Reply:
Using the first container in a group is the standard since many container groups have only 1 container it will default to that.

Reply:
That makes sense. Alternatively, you could make it the default *if* there's only one.  If there's more than one, you could either:
1) fail and require the user specify one (makes sense if this could be part of a script)
2) prompt the user to specify one (makes sense if this is part of an interactive flow)

I'm not sure which scenarios are supported here, so I can't say which would be better. However, arbitrarily choosing the first if there are more than one might surprise/upset users.